src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+#Base application info
+info.app.name=T2D-file-tracking
+info.app.description=T2D file tracking application
+info.app.version=@project.version@
+
+# Datasource configuration
+#spring.datasource.url =
+#spring.datasource.username =
+#spring.datasource.password =
+#spring.datasource.driver-class-name=org.postgresql.Driver
+#spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+
+storage.location=
+
+# Show or not log for each sql query
+spring.jpa.show-sql = true
+
+# Hibernate storageProperties
+## Use spring.jpa.storageProperties.* for Hibernate native storageProperties (the prefix is
+## stripped before adding them to the entity manager)
+## Hibernate ddl auto (create, create-drop, update, validate)
+spring.jpa.hibernate.ddl-auto=create-drop
+
+# Server Server HTTP port
+server.port= 9090
+# Prefix of the application path (eg. http://localhost:8082/test/files). Will work with spring boot 1.5 and not with 2.0
+server.contextPath= /test
+
+spring.http.multipart.max-file-size=128KB
```

Comment:
We are pretty much screwed if we only accept 128KB files :stuck_out_tongue: Can we set a more appropriate default?